tools/filterwriter/filterwriter.go

```diff
@@ -52,7 +54,12 @@ func (w *Writer) Write(p []byte) (int, error) {
 	w.chunk = chunk
 	if len(chunk) > 0 {
 		// we have remaining bytes, do not swallow them
-		time.AfterFunc(10*time.Second, func() {
+		if w.lastTimer != nil {
+			w.lastTimer.Stop()
+			w.lastTimer.C = nil
+			w.lastTimer = nil
```

Comment:
i think no need for this line. the new assignment will have the same effect (e.g.: there is no remaining reference for the object and garbage collector will clear it)

Reply:
true